did-sdk/src/index.ts

```diff
@@ -18,15 +17,10 @@ export async function resolve(didUri: string): Promise<ReadOnlyDidDocAPI> {
   throw new Error(`Unsupported did method (${didMethod}) ${didUri}`)
 }
 
-export async function createDidWebServer(port = 9999): Promise<DidWebServer> {
```

Comment:
Did you mean to remove this? That's fine if so, you just have other code in this PR that still seems to use it.

Reply:
I did. I switched it out for a static method. I guess I forgot to rebuild so never got an error. oops